web/pages/Products.jsx

```diff
@@ -0,0 +1,317 @@
+import { useState, useEffect } from 'react'
+import { useNavigate } from 'react-router-dom'
+import './css/Products.css'
```

Comment:
Aight, this looks like a good place for us to experiment with css modules.

https://vitejs.dev/guide/features#css-modules
https://create-react-app.dev/docs/adding-a-css-modules-stylesheet/